src/net_processing.cpp

```diff
@@ -1058,6 +1064,8 @@ class PeerManagerImpl final : public PeerManager
 
     void AddAddressKnown(Peer& peer, const CAddress& addr) EXCLUSIVE_LOCKS_REQUIRED(g_msgproc_mutex);
     void PushAddress(Peer& peer, const CAddress& addr) EXCLUSIVE_LOCKS_REQUIRED(g_msgproc_mutex);
+
```

Comment:
In the commit message of a0edbded5f `validation: log new header from unsolicited block`: `s/only only/only/`